### DISCUSSION TITLE
practices/do_not_run_out_of_steam/

### DISCUSSION THREAD

**Discussion by giscus** [sddevelopment-be/penguin-pragmatic-patterns#78] [Pattern Comments]
# practices/do_not_run_out_of_steam/

The pattern introduces the STEAM framework, a mental model for decision-making that encourages individuals to evaluate 
their options based on five key resources: Support, Time, Energy, Attention, and Money. 
This model is intended to help individuals make more informed and balanced decisions by considering the availability and impact of these resources.

https://patterns.sddevelopment.be/practices/do_not_run_out_of_steam/

**Comment by JasperVandenbrande** [comment]
Nice informative text that provides examples and solutions. I like the great visuals!
